packages/react-impression/src/components/Select/README.md

```diff
@@ -0,0 +1,112 @@
+### Examples
+
+**default**
+
+```js
+selectChangeHandle = (val, text) => {
+  Notification.info({
+    title: 'Select',
+    message: `${text} 被选中了！！！`,
+  })
+
+  this.refs.select3.setValue(null)
```

Comment:
[ref in example](https://react-styleguidist.js.org/docs/cookbook.html#how-to-use-refs-in-examples)